src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2552,11 +2691,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
-    Friend Partial Class BoundAggregateClause
+    Partial Friend Class BoundAggregateClause
         Protected Overrides Function ExpressionKind() As OperationKind
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
+            Get
+                Return ImmutableArray.Create(Of IOperation)(Me.CapturedGroupOpt, Me.GroupPlaceholderOpt, Me.UnderlyingExpression)
```

Comment:
>Me.GroupPlaceholderOpt [](start = 81, length = 22)

I think GroupPlaceholderOpt should be excluded from the list, it simply provides identity.